### PR TITLE
Smaller search, clearer call to action.

### PR DESCRIPTION
**Pull request by longhotsummer**
@xybrnet

### CODE REVIEW

File: pmg/templates/_header.html [hunk 1]
```diff
@@ -55,12 +55,12 @@
...
 
-      <div class="col-xs-12 col-md-5 col-lg-6">
+      <div class="col-xs-12 col-md-4 col-lg-5">
         <form class="navbar-form" role="search" action="/search">
           <div class="input-group">
-            <input type="text" class="form-control" name="q" placeholder="Search for committee meetings, bills, ministers, schedules and documents" value="{{q}}">
+            <input type="text" class="form-control" name="q" placeholder="What are you looking for?" value="{{q}}">
             <span class="input-group-btn">
-              <input type="submit" class="btn btn-success" onclick="form.submit()" value="Search" />
+              <button type="submit" class="btn btn-success"><i class="fa fa-search"></i> Search</button>
```
Comment (xybrnet) [inline pmg/templates/_header.html]: What do you think about only displaying the search icon, with no text?

Comment (longhotsummer) [inline pmg/templates/_header.html]: I thought about that, and other sites do it, but I think being explicit is better. If Font Awesome doesn't load, then the user can't see what that button does at all.

Comment (xybrnet) [inline pmg/templates/_header.html]: Cool.

**Comment by xybrnet** [comment]
:+1:

**Pull request merged**